index.html

```diff
@@ -57,14 +70,44 @@
         }
 
         .description {
-            font-weight: 500;
+            font-weight: 400;
+            font-size: 1.3rem;
             margin-bottom: 5px;
+            color: #424242;
```

Comment:
Modified the color of the .description class and h1 elements to a darker gray for improved readability.